docs/proposals/policy.md

```diff
@@ -0,0 +1,389 @@
+# Policy
+
+## Problem
+Policy extends the existing authorization rules in Kubernetes (https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/authorization.md) to fulfill requirements around extended verb sets, verb specific attributes, role based management of attribute based access control, and more efficient evaluation of authorization status at larger scales.  For discussion about verbs, see: https://github.com/GoogleCloudPlatform/kubernetes/issues/2877.  For discussion about allowing mutation of some fields in a resource and not others, see: https://github.com/GoogleCloudPlatform/kubernetes/issues/2726.
+
+## Use Cases
+ 1.  I want to apply the same set of policy rules to large numbers of projects.
+ 1.  I want to change the policy rules for a role and have it applied to all projects using that role.
+ 1.  I want to have policy changes take effect quickly, without any down time.
+ 1.  I want to have distinct policy rules to control group membership, project metadata, policy rules for roles, and policy rules for users.
+ 1.  OpenShift will ship with a set of default roles: view, edit, and admin
```

Comment:
Frame this a bit different.  For example, OpenShift believes there are three fundamental roles within namespaces, so we would implement a default policy with a view role, an edit role, and an admin role.  The admin policy within a namespace would allow everything except changing the quota or limits on the namespace (or the policy doc itself), the edit role would allow changing everything except role bindings on the policy or deleting the namespace, and the view role would confer read access on non sensitive data.